dcgm/tests/test_dcgm.py

```diff
@@ -0,0 +1,69 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+from unittest import mock
+
+import pytest
+
+from datadog_checks.dcgm import DcgmCheck
+
+
+@pytest.fixture
+def instance():
+    return {"openmetrics_endpoint": "http://localhost:9400/metrics"}
+
+
+@pytest.fixture
+def check(instance):
+    return DcgmCheck('dcgm.', {}, [instance])
+
+
+@pytest.fixture()
+def mock_metrics():
+    f_name = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'metrics.txt')
```

Comment:
By convention we put files like `metrics.txt` into a folder named `fixtures` inside the `tests` folder. Could you do the same here please?

Reply:
For sure, just updated it ✅